src/interfaces/dissolution-filing-interface.ts

```diff
@@ -19,6 +19,7 @@ export interface DissolutionFilingIF {
     legalType: string
     legalName: string
     identifier: string
+    foundingDate: string
```

Comment:
The next time we're in this area of code, we should declare this business block of type `BusinessIF`.